src/app/common/DataOverview.vue

```diff
@@ -78,55 +78,55 @@
             <TagList :tags="rowValue" />
           </template>
 
-          <template #name="{ row, rowValue }">
+          <template #entity="{ rowValue, row }">
```

Comment:
**Comment**: For every use of `DataOverview`, I now made sure that the table row data includes an `entity` object which is the raw resource object. Paired with it, there is also always a `detailViewRoute` which is a route location for the entity.

That allows every list view to easily link to a corresponding detail view by providing `detailViewRoute` in the table data.

For other linked cells, I couldn’t think of a better way than the slotted content that follows below (i.e. one for mesh, service, and zone).